CHANGELOG.md

```diff
@@ -32,11 +32,10 @@
 
 [comment]: <> (  - [nothing yet])
 
-[comment]: <> (### BUG FIXES)
-
-[comment]: <> (- [ibc])
+### BUG FIXES
 
-[comment]: <> (  - [nothing yet])
+- [ibc]
+  - Overflow if `ctx.host_current_height().revision_height < ctx.host_chain_history_size()` ([#685])
```

Comment:
Nit: Would be nice to have a slightly higher level description in the changelog.

```suggestion
  - Fix overflow bug in ICS03 client consensus height verification method ([#685])
```